fe/src/pages/index.vue

```diff
@@ -1,6 +1,9 @@
 <template>
-  <div>
-    WELCOME TO VUE!
-    <BaseDropdown />
+  <div class="h-screen grid place-content-center">
+    <button class="btn-gradient">Connect Wallet</button>
```

Comment:
Let's make use of the button component. (make sure to rebase from latest PR - feature/sol-14 to have the component available).